src/main/java/com/wiley/autotest/selenium/elements/upgrade/Window.java

```diff
@@ -11,6 +12,9 @@
 
     void switchToLast();
 
+    void switchToLastAfter(Actions action);
+    void switchToLastAfter(TeasyElement element);
```

Comment:
i would prefer keeping Window api straightforward and clean. Window represents a browser window and keeping it independent from anything like after click, after action is a plus from my point of view.
However, I like the idea of adding such layer of actions+conditions. Lets discuss in skype a potential place where we can have such layer of actions.